mode.go

```diff
@@ -44,7 +44,10 @@ func init() {
 
 func SetMode(value string) {
 	switch value {
-	case DebugMode, "":
+	case "":
+		ginMode = debugCode
+		value = DebugMode
+	case DebugMode:
```

Comment:
revert this part and add the following code. 

find

```
modeName = value
```

add before

```go
if value == "" {
    value = DebugMode
}
modeName = value
```

Reply:
done